src/unit_tests/os_auth/test_auth_validate.c

```diff
@@ -278,71 +276,71 @@ static void test_w_auth_replace_agent_force_disabled(void **state) {
     free_keyentry(&key);
 }
 
-static void test_w_auth_replace_agent_not_comply_antiquity(void **state) {
+static void test_w_auth_replace_agent_not_disconnected_long_enough(void **state) {
     w_err_t err;
     keyentry key;
     keyentry_init(&key, NEW_AGENT1, AGENT1_ID, NEW_IP1, NULL);
 
-    // Mocking antiquity
-    will_return(__wrap_OS_AgentAntiquity, 10);
-    config.force_options.connection_time = 100;
+    // Mocking disconnected_time
+    will_return(__wrap_get_time_since_agent_disconnection, 10);
+    config.force_options.disconnected_time_enabled = true;
+    config.force_options.disconnected_time = 100;
 
-    expect_string(__wrap__minfo, formatted_msg, "Agent '001' doesn't comply with the antiquity to be removed.");
+    expect_string(__wrap__minfo, formatted_msg, "Agent '001' has not been disconnected long enough to be replaced.");
```

Comment:
`"Agent '001' has not been disconnected long enough to be replaced."`
and
`"Agent '001' doesn't comply with the registration time to be removed."`
are from a previous task, but we should standardize these two similar messages. 

Like:
`"Agent '001' has not been disconnected long enough to be replaced."`
and
`"Agent '001' has not been registered long enough to be removed."`

or

`"Agent '001' doesn´t comply with the disconnection time to be replaced."`
and
`"Agent '001' doesn't comply with the registration time to be removed."`

Reply:
Fixed!